### DISCUSSION TITLE
Is there a "reset to default" method for Lanbon L8

### DISCUSSION THREAD

**Discussion by magicjohansson** [HASwitchPlate/openHASP#410] [Hardware]
I have somehow managed to upload a configuration-file to my L8 that prevents it from booting up, I only get a grey screen.
Is there a way to reset it to default or do I have to flash it again with the firmware ?

Thanks

**Comment by fvanroie** [comment]
There is a Factory Reset command that you can use from the serial console or Web UI.
If it fails to boot to the serial console and/or Web UI, you will need to reflash the device with the Erase Flash option...
